contracts/StakingSet.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.4.23;
+
+import "./interfaces/ValidatorSet.sol";
+import "./StakeBank.sol";
+
+contract StakingSet is ValidatorSet {
+  StakeBank stakeBank;
```

Comment:
how is this accessed?

Reply:
Hey, yes a lot of this is work in progress. I have some ideas on how to restructure, but I wanted to get some updates in and see if I can prepare it more for the build automation part